### PR TITLE
{lang}[foss/2024a,gfbf/2024a] R v4.4.2, R-bundle-CRAN v2024.11

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

* [x] #21907
* [x] #21905
* [x] #21906

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3515
**SUCCESS**
Build succeeded for 14 out of 14 (2 easyconfigs in total)
bear-pg0210u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8480CL (sapphirerapids), Python 3.6.8
See https://gist.github.com/branfosj/f413e3115ae12ebfa06d954268fba72c for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 11 out of 11 (2 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/e63cfb4b72285a76048bc26ff7af34e0 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3515
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/423e6e5aee40aa9094f829c961ff0a3c for a full test report.

**Comment by verdurin** [comment]
@branfosj it's complaining about clashing `Java` deps, with `NCO`.

File: easybuild/easyconfigs/r/R/R-4.4.2-gfbf-2024a.eb [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+    ('ncurses', '6.5'),
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.4.5'),
+    ('zlib', '1.3.1'),
+    ('SQLite', '3.45.3'),
+    ('PCRE2', '10.43'),
+    ('libpng', '1.6.43'),  # for plotting in R
+    ('libjpeg-turbo', '3.0.1'),  # for plottting in R
+    ('LibTIFF', '4.6.0'),
+    ('Java', '11', '', SYSTEM),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R/R-4.4.2-gfbf-2024a.eb:33]:
```suggestion
    ('Java', '17', '', SYSTEM),
```

**Comment by lexming** [comment]
@boegelbot please test @ generoso
CORE_CNT=16
EB_ARGS="--experimental --parallel-extensions-install"

File: easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.11-foss-2024a.eb [hunk 2]
```diff
@@ -0,0 +1,3508 @@
...
+    ('goldilocks', '0.3.0', {
+        'checksums': ['94df7f9d3550dc3790ea3c8f3f364f7131c907a235f498473c8c4e4f959b26c0'],
+    }),
+    ('r2rtf', '1.1.1', {
+        'checksums': ['d64b88187b5eb7bc69eae37f493fdae8417d5127d0a44f1586a00827bc321f52'],
+    }),
+    ('gsDesign', '3.6.5', {
+        'checksums': ['c5a8248708e4803b28cd33189963eb5dd1624df10ecf90f4c8ef7895cc022c0b'],
+    }),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R-bundle-CRAN/R-bundle-CRAN-2024.11-foss-2024a.eb:3499]:
```suggestion
    ('gcmr', '1.0.3', {
        'checksums': ['75773d5119179f186467d33608e9ce1c030d0755dfd6ff196546884078e89754'],
    }),
    ('frbs', '3.2-0', {
        'checksums': ['48505d415e3687afcb837e162b63b74173b3cd5df5f96140240e441c86343b2e'],
    }),
]
```

Sync with recent updates to `R-bundle-CRAN-2024.06-foss-2023b.eb`

**Comment by lexming** [comment]
@branfosj test failed with
>  Missing required dependencies for abind are not going to be installed: methods, utils

**Comment by branfosj** [comment]
> @branfosj test failed with
> 
> > Missing required dependencies for abind are not going to be installed: methods, utils

That is caused by `--parallel-extensions-install` which does not support detecting items that are provided via another module.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen3

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.10, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/665a9779b930ad2ee5e8fa64655c1d2c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Forgot to say thanks @branfosj !

**Pull request merged**